external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -136,6 +137,20 @@ class KafkaTestUtils(
     kdcConf.setProperty(MiniKdc.DEBUG, "true")
     kdc = new MiniKdc(kdcConf, kdcDir)
     kdc.start()
+    val krb5Conf = Source.fromFile(kdc.getKrb5conf, "UTF-8").getLines()
+    val rewriteKrb5Conf = krb5Conf.map(s => if (s.contains("libdefaults")) {
+      s + "\n" +
+        "    default_tkt_enctypes=aes128-cts-hmac-sha1-96\n" +
+        "    default_tgs_enctypes=aes128-cts-hmac-sha1-96 "
```

Comment:
Maybe we should first customize `krb5.conf` to test Kafka delegation token: https://issues.apache.org/jira/browse/SPARK-30037 @gaborgsomogyi 
Add then add these two configurations to `krb5.conf`.

Reply:
Check Kafka's `MiniKdc`,  it rewrite from hadoop-2.7's MiniKDC, and make `krb5.conf` customize. 
Add spark's own MiniKDC like Kafka is better. This is also beneficial to future work about kerberos test case.

Reply:
Hardcoding such things makes me worry, is it possible to ask whether it's supported or not?

Reply:
> Hardcoding such things makes me worry, is it possible to ask whether it's supported or not?

https://github.com/apache/spark/pull/26594#discussion_r350549930

Checked that `aes128-cts-hmac-sha1-96` is supported in jdk8 & jdk11.
Root cause is if we don't config this, jdk11's client will use `buildin` enctype of jdk11